convene-web/app/lib/furniture/markdown_text_block/_in_room.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="card">
+  <%= furniture.to_html.html_safe %>
```

Comment:
I know why it's safe here but do you think we should add `Redcarpet` is making it safe here?